lib/Kind.py

```diff
@@ -1,86 +1,92 @@
-import common
-import time
-import os
+# This file is depricated use ClusterProvider.py instead
```

Comment:
FYI: This file is being deprecated so commented as of now until we completly port it to cluster provider bash script.